tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1800,12 +1800,9 @@ schedule(void)
           (void)cons_vec_always_metadata();
         }
         if ((check_for_loop_directive(ILT_LINENO(ilt), 191, 0x4))) {
```

Comment:
It seems that this condition can be merged into the condition of the previous block, since the blocks are the same?

Reply:
@pawosm-arm ping?

Reply:
done